package.json

```diff
@@ -48,6 +48,7 @@
     "@storybook/addons": "^5.1.0-alpha.27",
     "@storybook/react": "^5.1.0-alpha.27",
     "@types/classnames": "^2.2.7",
+    "@types/express": "^4.16.1",
```

Comment:
Nop, as long as we use express only in `bierzo-wallet`, this type should remain in `bierzo-wallet` project 😉